tests/cypress/e2e/teacher_page/class_page/logs_button.cy.js

```diff
@@ -0,0 +1,22 @@
+import {loginForAdmin, loginForTeacher} from '../../tools/login/login.js'
+import {createClass} from '../../tools/classes/class.js'
+
+
+describe('Is able to go back to teacher page', () => {
+  it('Passes', () => {
+    loginForTeacher();
+    cy.wait(500);
+    createClass();
+    cy.get(":nth-child(1) > :nth-child(3) > .no-underline").click(); // Press view class button
+
+    var currentUrl = '';
+    cy.url().then(url => {
+      currentUrl = url;
+      let logsButton = `[onclick="window.location.href = '/logs/class/` + currentUrl.substring(currentUrl.indexOf('class/')+6) + `'"]`; // Get logs button

```

Comment:
Same here, what is we decide to change the url of the class? Is there a way to make this more flexible as well?